shell/common/node_bindings.cc

```diff
@@ -394,15 +411,33 @@ node::Environment* NodeBindings::CreateEnvironment(
   }
 
   if (browser_env_ == BrowserEnvironment::BROWSER) {
-    // SetAutorunMicrotasks is no longer called in node::CreateEnvironment
-    // so instead call it here to match expected node behavior
+    // This policy ensures microtask checkpoints are explicitly invoked.
```

Comment:
I don't think that's what this policy does...? Rather, it _doesn't_ automatically invoke any microtask checkpoints and instead leaves it up to the embedder to invoke them.

It might also be useful to mention that this is what Node does & thus is required by Node.

Reply:
you're right and is what i meant but mangled explaining - i'll clarify the phrasing :)